.travis.yml

```diff
@@ -55,3 +55,4 @@ install:
   - npm run compile
```

Comment:
This isn't super important at the moment, but it looks like `packages/boiler-task-babel/node_modules` should be added to the `cache` block in this file.